app/models/search/tag_search_form.rb

```diff
@@ -43,6 +45,12 @@ def set_fandoms
     @options[:fandom_ids] = Tag.where(name: names).pluck(:id)
   end
 
+  def set_wrangling_status
+    return unless @options[:canonical]
+    
+    @options[:wrangling_status] = "canonical"
```

Comment:
https://github.com/otwcode/otwarchive/blob/5748b65c170c976e33f7a71b5903da5c9797bdc9/app/views/tags/_search_form.html.erb#L42-L49

`@options[:wrangling_status] = "canonical"` only if `@options[:canonical]` is `T`. You should handle the `F` to `"noncanonical"` case as well.